### PR TITLE
made collection ops help sections slightly more readable

### PR DESCRIPTION
**Pull request by nekrut**
Collection ops have really bad help sections. This PR fixes that.

### CODE REVIEW

File: lib/galaxy/tools/unzip_collection.xml [hunk 1]
```diff
@@ -12,12 +12,19 @@
...
-    This tool will create new history datasets from your collection
-    but your quota usage will not increase.
-  </help>
+This tool takes a paired dataset collection and builds two datasets from it. If mapped over a list of paired datasets, this tool will produce two lists of datasets. 
+
+----
+
+**Example**
+
+If a collection consists of two forward and two reverse datasets (e.g., forward and reverse reads from a sequencing experiment) this tool will output two collection: one consisting of forward reads and one of reverse.
```
Comment (mvdbeek) [inline lib/galaxy/tools/unzip_collection.xml]: s/two collection/two collections/ and maybe `one of reverse reads` ?

**Comment by martenson** [comment]
If tou want this on Main it needs to target 17.01.@nekrut

**Review by mvdbeek — APPROVED** [review]

**Comment by jmchilton** [comment]
Thanks for the kind words Anton.

**Comment by jmchilton** [comment]
I've opened a backport of this to 17.01 with #3788 so it can land up on main.

**Pull request merged**